pkgs/development/tools/misc/autogen/default.nix

```diff
@@ -62,9 +61,13 @@ stdenv.mkDerivation rec {
       sed -e "s|$bin/bin|/no-such-autogen-bin-path|" -i $f
       sed -e "s|$lib/lib|/no-such-autogen-lib-path|" -i $f
     done
-  '';
 
-  #doCheck = true; # 2 tests fail because of missing /dev/tty
+    # remove /build/** from RPATHs
+    for f in "$bin"/bin/*; do
+      local nrp="$(patchelf --print-rpath "$f" | sed -E 's@(:|^)/build/[^:]*:@\1@g')"
```

Comment:
This is problematic for darwin https://hydra.nixos.org/build/122123361

Reply:
Oh, right, I think this isn't the first time I forgot about patchelf on darwin.  I can have a look, in about 12h, probably.

Reply:
potential fix https://github.com/NixOS/nixpkgs/pull/90284